### PR TITLE
A bunch of EU resorts and one japanese resort.

### PR DESCRIPTION
**Pull request by PapeSnowiNavi**
First attempt at this Pull request stuff, hope that it is right. :)

### CODE REVIEW

**Comment by FezVrasta** [comment]
@pirxpilot ?

**Comment by pirxpilot** [comment]
Thanks for the PR. Sorry it took me so long to review it.

I am going to add some general remarks here - and also add some comment and suggestions inline later on.

- it makes things easier for me if PRs contain one resort or one type of resorts at a time - in this way if there is a problem with one resort it does not block another - a separate branch for each newly added resort and a PR based off that branch makes things much easier; obviously resorts that share the same parser can and should share a PR
- resort name in the descriptor is something that users see in the UI, resort `slug` (the name of the directory in which the resort.json resides) is a part of the URL and liftie is always using [kebab-case] for it - so `alta-badia` and not `Alta-Badia` and `les-4-valles` and not `les_4_valles`
- if several resorts have the same parser the common code should end up in `/lib/tools`  - see for example how `/lib/tools/vail.js` is used in the code
- `ll` is not optional and it cannot be set to `null` - location is used be weather forecast and - where available - webcams
- please remove the code that you are removing instead of commenting out the old code and adding the new one 
- make sure that the tests are passing before submitting PR

If you can submit a single resort PR we can work out the kinks and land it in liftie. If not I'll try to cherry pick from what I see here. Thanks again.

[kebab-case]: https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case

**Comment by JeppePape** [comment]
> Thanks for the PR. Sorry it took me so long to review it.
> 
> I am going to add some general remarks here - and also add some comment and suggestions inline later on.
> 
> * it makes things easier for me if PRs contain one resort or one type of resorts at a time - in this way if there is a problem with one resort it does not block another - a separate branch for each newly added resort and a PR based off that branch makes things much easier; obviously resorts that share the same parser can and should share a PR
> * resort name in the descriptor is something that users see in the UI, resort `slug` (the name of the directory in which the resort.json resides) is a part of the URL and liftie is always using [kebab-case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case) for it - so `alta-badia` and not `Alta-Badia` and `les-4-valles` and not `les_4_valles`
> * if several resorts have the same parser the common code should end up in `/lib/tools`  - see for example how `/lib/tools/vail.js` is used in the code
> * `ll` is not optional and it cannot be set to `null` - location is used be weather forecast and - where available - webcams
> * please remove the code that you are removing instead of commenting out the old code and adding the new one
> * make sure that the tests are passing before submitting PR
> 
> If you can submit a single resort PR we can work out the kinks and land it in liftie. If not I'll try to cherry pick from what I see here. Thanks again.

I see, thanks for the information :)

I was thinking that maybe the whole of the "interwebs" domain could be made into a tool ? under the "tools" folder, similarily to how boyne, aspen and intrawest are. As the way each resort is setup is obv the same and adds a 60-100 ~ areas in especially Austria, but also some other european and international places ?

Then the other ones could be added individually like requested ?

What do you think ?

**Comment by pirxpilot** [comment]
> I was thinking that maybe the whole of the "interwebs" domain could be made into a tool ? under the "tools" folder, similarily to how boyne, aspen and intrawest are. As the way each resort is setup is obv the same and adds a 60-100 ~ areas in especially Austria, but also some other european and international places ?
> 
> Then the other ones could be added individually like requested ?
> 
> What do you think ?

Yup. That would be my preference as well.

File: lib/resorts/Matterhorn/index.js
Comment (pirxpilot) [inline lib/resorts/Matterhorn/index.js:1]: this does not seem to work any more -- probably the page has been changed recently

**Comment by pirxpilot** [comment]
I am pretty sure that all new resorts that were included here (not based on `intermaps`) are now released. I am going to close this PR now. Please open new/separate PRs once you have more fixes or changes ready. Thanks again!

**Pull request closed without merging**